src/app/mat-quill/mat-quill-base.ts

```diff
@@ -96,16 +99,34 @@ export abstract class _MatQuillBase
       this.ngControl.valueAccessor = this;
     }
 
-    this.onBlur.subscribe(() => {
+    this.onContentChanged
+      .pipe(
+        takeUntil(this.unsubscribeAll),
+        debounceTime(300),
+        distinctUntilChanged()
+      ).subscribe(() => {
```

Comment:
please fix indention :)

why introducing `unsubscribeAll`? just store the focus, blur and contentChange subscription on class properties and unsubscribe in ngOnDestroy.

Reply:
ok, I'm going to fix indention. 

And on the other hand, I introduce the subject `unsubscribeAll` to unsubscribe observables because i thought that feel more easy to handle, but i will change it, thanks you.

Reply:
@KillerCodeMonkey, changes are done, can you tell me why you prefeer have three `subscription's` stored vs only one `subject` in the `pipe`? 

Regards!

Reply:
because you are creating the subscriptions anyways when your are calling `subscribe`. But creating a Subject that is only emitted once is a little bit strange..

you could even create an array of subscriptions and push all subscriptions on it and loop over it + unsubscribe in ngOnDestroy ;-)

Reply:
thanks you @KillerCodeMonkey, you're right, is a little bit strange emit only once.